app/models/ability.rb

```diff
@@ -45,6 +45,11 @@ def not_signed_in
     # can view Commercials of confirmed Events
     can :show, Commercial, commercialable_type: 'Event', commercialable_id: Event.where(state: 'confirmed').pluck(:id)
     can [:show, :create], User
+
+    can [:new, :create], Event do |event|
```

Comment:
But with ichain we do **not** create users from within OSEM.
So these lines are only valid when iChain is **off**. Because those lines refer to not_signed_in user, in which case the new action of proposal form includes fields for user account creation.